blocks/PersonList/PersonList.tsx

```diff
@@ -31,79 +31,71 @@ export interface PersonListProps extends BlockFields {
-            {person.contact && (
-              <ContactPoints>
-                {person.contact.email && (
-                  <ContactPoint href={`mailto:${person.contact.email}`}>
-                    {person.contact.email}
-                  </ContactPoint>
+      <Row rowGap={2.5}>
+        {props.content.map(
+          ({ name, nickname, image, comment, contact }, index) => {
+            const firstName = name.split(" ").slice(0, 1).join(" ")
+            return (
+              <Person col={6} m={4} l={3} key={index}>
+                <PersonImage mask={randomCircle(index).src} image={image} />
+                <div>
+                  <Name>{nickname ?? firstName}</Name>
+                  <Alias>{name}</Alias>
+                </div>
+                {comment && <SubTitle>{comment}</SubTitle>}
+                {contact && (
+                  <ContactPoints>
+                    {contact.email && (
+                      <ContactPoint href={`mailto:${contact.email}`}>
+                        {contact.email}
+                      </ContactPoint>
+                    )}
+                    {contact.phone && (
+                      <ContactPoint href={`tel:${contact.phone}`}>
+                        {contact.phone}
+                      </ContactPoint>
+                    )}
+                  </ContactPoints>
                 )}
-                {person.contact.phone && (
-                  <ContactPoint href={`tel:${person.contact.phone}`}>
-                    {person.contact.phone}
-                  </ContactPoint>
-                )}
-              </ContactPoints>
-            )}
-          </Person>
-        ))}
-      </PersonListContainer>
+              </Person>
+            )
+          }
+        )}
+      </Row>
     </Container>
   </Block>
 )
 
-const PersonListContainer = styled.div`
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  gap: 3em 2em;
-
-  @media ${max("l")} {
-    grid-template-columns: 1fr 1fr 1fr;
-  }
-
-  @media ${max("m")} {
-    grid-template-columns: 1fr 1fr;
-  }
-
-  @media ${max("s")} {
-    grid-template-columns: 1fr;
-  }
-`
-
-const Person = styled.div`
+const Person = styled(Column)`
   display: flex;
+  gap: 0.5em;
   flex-direction: column;
-  gap: 0.75rem;
+  justify-content: flex-start;
+  text-align: center;
 `
 
 const PersonImage = styled(Image)`
+  width: 60%;
   height: auto;
-  width: 100%;
+  align-self: center;
   background-color: ${theme.color.lightAccent};
   mask-image: url("${(props: { mask: string }) => props.mask}");
   mask-size: 100% 100%;
   mask-position: center;
   mask-repeat: no-repeat;
-  align-self: center;
 `
 
 const Name = styled(Header3)`
-  margin: 0.8rem 0 0;
-  padding: 0;
+  margin: 0;
+`
+
+const Alias = styled(SmallText)`
+  color: ${theme.color.darkAccent};
+  margin: 0;
 `
 
 const SubTitle = styled(Text)`
   margin: 0;
-  line-height: 1.2em;
 `
 
 const ContactPoints = styled.div`
```

Comment:
Tohle mi připadá zbytečné, má to téměř stejné nastavení, jako rodič (Parent) a když to odstraním, tak to nemá téměř žádný vliv na vzhled. Akorát to komplikuje kód.